config/serverless.oblt.yml

```diff
@@ -6,6 +6,7 @@ xpack.cloudSecurityPosture.enabled: false
 xpack.infra.enabled: true
 xpack.uptime.enabled: true
 xpack.securitySolution.enabled: false
+xpack.search.notebooks.enabled: false
```

Comment:
We want it enabled in `serverless.es.yml` which is will be by default. But since `serverless.oblt` & `serverless.security` don't have the persistent console enabled then enabling this plugin is a waste.